src/matrix/kernelkroneckermat.jl

```diff
@@ -14,7 +14,7 @@ where `D` is given by `dims`.
 
 !!! warning
 
-    Require `Kronecker.jl` and for `iskroncompatible(κ)` to return `true`.
+    Requires `Kronecker.jl` and for `iskroncompatible(κ)` to return `true`.
```

Comment:
Unrelated change it seems?
```suggestion
    Require `Kronecker.jl` and for `iskroncompatible(κ)` to return `true`.
```

Reply:
would you like this in a separate PR ?